Flea/Shared/MainLayout.razor.css

```diff
@@ -5,7 +5,8 @@
 }
 
 .main {
-    flex: 1;
+    background-color: #0d0d0d;
+    width: -webkit-fill-available;
```

Comment:
Probably not the solution you are looking for.
You generally want to avoid (or at least provide fallback values for) vendor prefixed (e.g. `-webkit-*`, `-moz-*`, etc.) css properties/values since they have very limited browser compatibility and sometimes do not conform to standards.

(This does not work in Firefox 😉)

`flex:1;` with undefined `width` is generally preferred (since the surrounding container is a flexbox), but I assume that it was causing some issue since you changed it?

Otherwise you can try `width: calc(100vw - 64px);`.

Reply:
Fixed